code/game/gamemodes/cult/ritual.dm

```diff
@@ -165,522 +10,269 @@ var/engwords = list("travel", "blood", "join", "hell", "destroy", "technology",
+
+	text += "<font color='red'><b>Wraith</b></font><br>The Wraith is a fast, lethal melee attacker which can jaunt through walls. However, it is only slightly more durable than a shade.<br><br>"
+
+	text += "<font color='red'><b>Juggernaut</b></font><br>The Juggernaut is a slow, but durable, melee attacker which can produce temporary forcewalls. It will also reflect most lethal energy weapons.<br><br>"
+
+	text += "<font color='red'><b>Artificer</b></font><br>The Artificer is a weak and fragile construct, able to heal other constructs, produce more <font color='red'><b>soulstones</b></font> and <font color='red'><b>construct shells</b></font>, \
+	construct fortifying cult walls and flooring, and finally, it can release a few indiscriminate stunning missiles.<br><br>"
+
+	text += "<font color='red'><b>Harvester</b></font><br>If you see one, know that you have done all you can and your life is void.<br><br>"
+
+	var/datum/browser/popup = new(user, "tome", "", 800, 600)
+	popup.set_content(text)
+	popup.open()
+	return 1
+
+/obj/item/weapon/tome/proc/scribe_rune(mob/living/user)
+	var/turf/Turf = get_turf(user)
+	var/chosen_keyword
+	var/obj/effect/rune/rune_to_scribe
+	var/entered_rune_name
+	var/list/possible_runes = list()
+	var/list/shields = list()
+	if(locate(/obj/effect/rune) in Turf)
+		to_chat(user, "<span class='cult'>There is already a rune here.</span>")
+		return
+	for(var/T in subtypesof(/obj/effect/rune) - /obj/effect/rune/malformed)
+		var/obj/effect/rune/R = T
+		if(initial(R.cultist_name))
+			possible_runes.Add(initial(R.cultist_name)) //This is to allow the menu to let cultists select runes by name rather than by object path. I don't know a better way to do this
+	if(!possible_runes.len)
+		return
+	entered_rune_name = input(user, "Choose a rite to scribe.", "Sigils of Power") as null|anything in possible_runes
+	if(!Adjacent(user) || !src || qdeleted(src) || user.incapacitated())
+		return
+	for(var/T in typesof(/obj/effect/rune))
+		var/obj/effect/rune/R = T
+		if(initial(R.cultist_name) == entered_rune_name)
+			rune_to_scribe = R
+			if(initial(R.req_keyword))
+				var/the_keyword = stripped_input(usr, "Please enter a keyword for the rune.", "Enter Keyword", "")
+				if(!the_keyword)
+					return
+				chosen_keyword = the_keyword
+			break
+	if(!rune_to_scribe)
+		return
+	Turf = get_turf(user) //we may have moved. adjust as needed...
+	if(locate(/obj/effect/rune) in Turf)
+		to_chat(user, "<span class='cult'>There is already a rune here.</span>")
+		return
+	if(!Adjacent(user) || !src || qdeleted(src) || user.incapacitated())
+		return
+	if(ispath(rune_to_scribe, /obj/effect/rune/narsie) || ispath(rune_to_scribe, /obj/effect/rune/slaughter))//may need to change this - Fethas
+
+		if(ticker.mode.name == "cult")

```

Comment:
how fix (i think you may have answered by i am going down before i check for updates...)